rega-genotype/src/main/java/rega/genotype/parasites/giardia/GiardiaTool.java

```diff
@@ -65,11 +65,12 @@ public void analyze(AbstractSequence s) throws AnalysisException {
     }
 
 	private boolean phyloAnalysis(PhyloClusterAnalysis pca, AbstractSequence s) throws AnalysisException {
+		
 		if (pca == null)
 			return false;
 		
 		PhyloClusterAnalysis.Result r = pca.run(s);
-
+		
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file